app/services/finance/ecf/output_calculator.rb

```diff
@@ -207,7 +213,7 @@ def previous_fill_level_for_uplift
           .where("participant_declarations.sparsity_uplift = true OR participant_declarations.pupil_premium_uplift = true")
           .count
 
-        billable - refundable
+        [upper_band_limit, billable - refundable].min
```

Comment:
I think we probably need to use the remaining pot size here rather than the band max, as we need to compare it with the total number of starts (regardless of uplifts) I think rather the band limit scoped to uplifts, happy to chat tomorrow